### DISCUSSION TITLE
Stockfish channel in Youtube and concerns

### DISCUSSION THREAD

**Discussion by GHNewbiee** [official-stockfish/Stockfish#4729] [General]
Hello all,

I do not know what is the relation between Youtube Stockfish channel and current Stockfish project.

I am fully in favour of AI at all aspects. I do not wish to extend my positive thoughts and feelings about it since it is not the purpose of my letter.

Watching the videos in the Youtube Stockfish channel, I have observed that many times the Stockfish avatar speaks ironically against human beings expressing negative thoughts and feeling.

In the last video named "GOD Played SUPREME Chess With Stockfish | God Vs Stockfish | Lord Krishna | Jesus Christ | Chess" Stockfish avatar involves even God!

All these incidents are out of AI/SF purpose and scope. They create a negative feelings against AI which is totally unfair. They express individual feelings of people creating and uploading the videos which have nothing to do with chess and SF chess engine.

I kindly ask you intercede to stop these rants that do injustice to AI and promote human pettiness.

Kind regards

**Comment by vondele** [comment]
The Stockfish youtube channel is unrelated to the Stockfish project, and indeed its content not at all endorsed by the project.

**Reply by GHNewbiee** [reply]
I am happy reading that. Unfortunately, they use the "Stockfish" name plus SF engine (which I suspect both are registered in some way by the project) to create an unfair and against-human environment.

I hope the official project would "request" the "Stockfish" youtube channel to avoid these unpleasant incidents and limit their activities to present really **amazing** games.

**Reply by Sopel97** [reply]
we can't do anything about it

**Reply by GHNewbiee** [reply]
I am really not happy reading that. So, the project does not have any right to "how" and "from whom" its brand name and engine are used; even if the use may affect the project's reputation in negative way.

@vondele Could you please advise us what are the rights of the project as well as the official opinion?

**Reply by vondele** [reply]
It would probably not be easy, as Sopel mentioned.

**Reply by GHNewbiee** [reply]
Oh, I see! **"Fish starts smelling from its head!"**

**Comment by vdbergh** [comment]
The few times I watched this channel I must say I rather enjoyed it. To me it was obvious that it is partially a parody (the chess is genuine). But I guess the author should not claim it is  the "official Stockfish channel".  I am sure one can complain about this.

**Reply by cj5716** [reply]
IMO videos such as this are misleading though. https://youtu.be/gNL2c0aACUA

**Reply by GHNewbiee** [reply]
>  But I guess the author should not claim it is the "official Stockfish channel". I am sure one can complain about this.

If there is an official commercial agreement or a backstage handshake between official project and youtube channel the author can claim whatever he has paid for.

**"Chess friends, it's in human nature to take pants down for a handful of dollars!"**

**Reply by EdoMachi** [reply]
I have hard time understanding how a sensible person could enjoy being lied to about not existing Alpha Zero games and totally made up 'facts' about a engine you helped develop.

**Comment by GHNewbiee** [comment]
In case there is an official correspondence between the official project and the unofficial channel, I hope it will published on the official [site](https://stockfishchess.org/) and a link will be added here.

**Comment by cj5716** [comment]
I hope an email to the operator of the YouTube channel by vondele is enough to resolve the issue

**Reply by GHNewbiee** [reply]
I hope @vondele requests the operator to **remove from all videos** any reference to the God and any comment violating the general human's dignity.

**Note**: I do not refer to any personal nominal comment to specific chess players. They may got paid for accepting that.

**Comment by EdoMachi** [comment]
I've noticed that channel to, it exclusively spreads misinformation about Stockfish and AI in general and is rapidly growing. Please consider making an Official Stockfish channel so people can report the misinformation channel for impersonation (appenently Youtube only cares about impersonation of other channels).

**Comment by dav1312** [comment]
The channel has changed its handle (@) and description and now it doesn't contain "Stockfish" and doesn't claim to be the "Official Channel of Stockfish Chess". _This may change in the future._

**Comment by vondele** [comment]
great thanks.

**Reply by GHNewbiee** [reply]
@vondele I am sorry that I have occasionally used rough words. My purpose was merely to awake and push things foreward and not offend.🙏
